CMakeLists.txt

```diff
@@ -24,14 +24,14 @@ IF (MSVC)
 	IF(MSVC_TOOLSET_VERSION STREQUAL 142)
 		SET(APACHE_DL_VS VS16)
 	ELSEIF(MSVC_TOOLSET_VERSION STREQUAL 141)
-		SET(APACHE_DL_VS VS15)
+		SET(APACHE_DL_VS VC15)
```

Comment:
Does `VS16` also need to change above, or just `VS15`?

Reply:
I've been wondering the same, it seems that they have changed the naming convention to use VS instead of VC from version 16 (https://www.apachelounge.com/viewtopic.php?t=8261)